### ISSUE TITLE
FR6.  Internal Clock System

### ISSUE THREAD

**Issue by fahmi009** [WSU-4110/LeftOvr#7]
|FR6.  Internal Clock System|
|---|
|**Goal**: The system counts down from 24 hours once the meal is confirmed to have been picked up
**Stakeholders:** Customers|
| **Description:** System lacks required tools, so unable to complete feature. Once a customer picks up the food they reserved, there will be a clock that counts down from 24 hours, within that time frame the user will not be able to pick up or reserve another meal until that timer is up.  | **Origin:** Based on the initial project specification document, team members came up with this description during the first meeting.|
| **Version:** 1.0  |
|**Date:** 09/28/2020|  **Priority:** 1 |

**Comment by White12487** [comment]
I found a way how we could implement this in the django framework. We have to import the celery api then connect it to our project. We do this by choosing and downloading a broker that can run the celery app in django, (online everyone says RabbitMQ is good), then we let it run in the background as we create a tasks file near the settings.py file then we literally just import celery and the file that we created the methods in, into the project. So, we will need an event or basically a sort of event listener for when a customer orders a meal to start the timer.
